lib/ansible/modules/windows/win_domain_group.py

```diff
@@ -234,4 +234,9 @@
   returned: group exists
   type: str
   sample: S-1-5-21-2171456218-3732823212-122182344-1189
+created:
+  description: Whether a group was created
+  returned: when a new group was created
```

Comment:
just a nitpick, this should be always as we return `$false` in the case of a group not being created. Same as the `win_domain_user` entry.